src/ejabberd_s2s.erl

```diff
@@ -262,7 +262,7 @@ code_change(_OldVsn, State, _Extra) ->
                To :: jid:jid(),
                Acc :: mongoose_acc:t(),
                Packet :: exml:element()) ->
-        done. % this is the 'last resort' router, it always returns 'done'.
+    {done, mongoose_acc:t()}. % this is the 'last resort' router, it always returns 'done'.
```

Comment:
Do we need return `{done, mongoose_acc:t()}.`? Maybe make sense return `mongoose_acc:t()`?

Reply:
To match with more efficiency, and without looking into the `mongoose_acc`. If we want to match on the return value of these functions and check for the successful case, pattern-matching on a tagged tuple is more efficient than into a map with a specific key inside, and also, we would be exposing the internals of such map.

Reply:
But this function always return the same, this mean that `mongoose_acc:t()` will always returned, and also this mean that we can check on `mongoose_acc:t()` in places where we need it. I do not insist, just was confusing of returning the same result always.

Reply:
Yep. Make sense.